### PR TITLE
Show webinar time and timezone with date

### PR DESCRIPTION
**Pull request by thomasthesecond**
The `formatString` method in Gatsby/GraphQL uses moment.js for formatting and it didn't seem to output the correct time. date-fns was added to provide better time formatting and conversion based on the UTC offset set in Contentful. The time on the webinar page will update to the user's current time zone.

To test, change the time zone in your computer's system preferences and refresh the page.

image

image

### CODE REVIEW

File: src/components/webinars/Webinar.js [hunk 1]
```diff
@@ -15,7 +16,11 @@ export default ({ webinar }) => (
       </div>
 
       <div className={styles.body}>
-        <p>{webinar.webinarDate}</p>
+        <p>
+          <time dateTime={webinar.webinarDate}>
+            {format(new Date(webinar.webinarDate), 'EEEE, MMMM d, yyyy, h:mm a')}
```
Comment (zkuzmic) [inline src/components/webinars/Webinar.js:21]: This format string is obviously working, but I don't really get it — `EEEE` isn't included in the `date-fns` docs and it looks like you may want an upper case `D`. 

Based on what I'm seeing on those docs, the string should be `dddd, MMMM D, YYYY, h:mm a`.

Also, I'm assuming marketing requested that you include the time, but I'm not sure it's helpful without a timezone attached. What do you think?

Comment (zkuzmic) [reply src/components/webinars/Webinar.js:21]: Here are the date-fns docs I was referencing btw
https://date-fns.org/v1.28.5/docs/format

Comment (thomasthesecond) [reply src/components/webinars/Webinar.js:21]: You linked to v1 docs; I'm using v2.

https://date-fns.org/v2.14.0/docs/format

Comment (thomasthesecond) [reply src/components/webinars/Webinar.js:21]: `EEEE` gives us the day of the week
`d` gives us day of the month; `1, 2, 3, 4, etc`

Comment (zkuzmic) [reply src/components/webinars/Webinar.js:21]: ah that makes sense, sorry about that

Comment (thomasthesecond) [reply src/components/webinars/Webinar.js:21]: As for the timezone, not sure I'm going to be able to reliably show the timezone considering daylight savings time. I can show the UTC offset, but I'm not sure if that's helpful or if most people understand it? I thought the fact that the time is converted based on the user's system time zone would be fine.

The other option I was considering is put a new field in Contentful for time and allow a string which we instruct to be entered in Pacific time and then we can hardcode it `July 21, 2020, 9:00 AM Pacific`. Everyone would see the time in Pacific. I wasn't sure if this was better or worse of having the time automatically change based on the user.

Comment (thomasthesecond) [reply src/components/webinars/Webinar.js:21]: date-fns has a timezone option for `format`, I'll try to add it.

Comment (zkuzmic) [reply src/components/webinars/Webinar.js:21]: I think an offset or always setting it to Pacific would be a little more clear. To me at least, it didn't feel obvious that the time was based on my computer's time zone. If date-fns has a clean way to format the user's local time zone, that seems like the best option to me.

Comment (thomasthesecond) [reply src/components/webinars/Webinar.js]: Added a separate field for time and a new field for timezone in Contentful.

image

**Comment by thomasthesecond** [comment]
Updated screenshot with new timezone data

image

File: src/components/webinars/Webinar.js (hunk 1 above)
Comment (thomasthesecond) [reply src/components/webinars/Webinar.js]: date-fns was removed; relying on Gatsby/GraphQL `formatString` method

**Pull request merged**